humanoid_league_transform/src/humanoid_league_transform/transformer.py

```diff
@@ -419,6 +424,11 @@ def _line_plane_intersections(self, plane_normal, plane_point, ray_directions):
 
         return ray_directions
 
+    def _check_if_not_bottom_of_the_image(self, position, thresh):
```

Comment:
I would prefer to name the method to something like `_object_at_bottom_of_image` (and refactor it to match the name) so that it is more clear what the return value `True` or `False` mean (I think `check_if_not_bottom_of_the_image` is rather confusing)